Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/RiskLevelRepository.kt

```diff
@@ -14,7 +14,6 @@ object RiskLevelRepository {
         MutableStateFlow(LocalData.lastSuccessfullyCalculatedRiskLevel().raw)
```

Comment:
Here we access "LocalData.lastSuccessfullyCalculatedRiskLevel().raw" directly whereas we use "getLastSuccessfullyCalculatedScore().raw" in line 10 
We should align this.